content/custom_resources/partials.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Partials"
+gh_repo = "chef-web-docs"
+
+[menu]
+  [menu.infra]
+    title = "Partials"
+    identifier = "chef_infra/cookbook_reference/custom_resources/partials"
+    parent = "chef_infra/cookbook_reference/custom_resources"
+    weight = 20
++++
+
+[InfoQ article](https://www.infoq.com/news/2020/05/chef-infra-16/)
+
+Resoruce partials are a way of DRYing up resource code.
```

Comment:
*Resource

Reply:
"DRYing" is a bit jargony for a general audience. Maybe something like "Resource partials are a way of allowing resources to share common code.  We recommend using partials any time you need to share code across three or more resources.

Reply:
Sounds good!